src/Features/Core/Portable/ExtractInterface/ExtractInterfaceCodeAction.cs

```diff
@@ -56,6 +56,6 @@ protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperation
             return operations;
         }
 
-        public override string Title => FeaturesResources.Extract_Interface;
+        public override string Title => FeaturesResources.Extract_interface;
```

Comment:
huh?

Reply:
I talked to Kendra after a comment Cyrus left about title casing vs. sentence casing. She said sentence casing was preferable to match the rest of VS code actions, so I went ahead and changed the casing for both extract method and extract interface.

Reply:
I talked to Kendra after a comment Cyrus left about title casing vs. sentence casing. She said sentence casing was preferable to match the rest of VS code actions, so I went ahead and changed the casing for both extract method and extract interface.